recipe/activate-gcc.sh

```diff
@@ -190,16 +193,16 @@ else
   fi
 
   # fix prompt for zsh
-  if [[ -n "${ZSH_NAME:-}" ]]; then
+  if [ -n "${ZSH_NAME:-}" ]; then
     autoload -Uz add-zsh-hook
 
     _conda_clang_precmd() {
-      HOST="${CONDA_BACKUP_HOST}"
+      export HOST="${CONDA_BACKUP_HOST}"
```

Comment:
IDK about this export. :/

Reply:
Can you elaborate?
I just pushed a commit to not change it. But it shouldn't be important either way since `HOST` is already exported by the activation script.

Reply:
Yeah I am just worried about changing this stuff too much. These changes are hard to test in all of the edge cases and so things are somewhat brittle.